test/unit/core/bit_manipulation_test.cpp

```diff
@@ -9,6 +9,8 @@
 
 #include <gtest/gtest.h>
 
+#include <sdsl/bits.hpp>
```

Comment:
if you include it here, we might not spot issues, if the header was not correctly included in the bit_manipulation file or removed. Also the header test should catch them, but I don't see the necessity to add this here.

Reply:
Ok, I just always include sdsl/bits now in the header